src/app/docs/selection-box/selection-box-docs.component.html

```diff
@@ -0,0 +1,269 @@
+<sky-docs-demo-page
+  moduleName="SkySelectionBoxModule"
+  moduleSourceCodePath="src/app/public/modules/selection-box/"
+  pageTitle="Selection box"
+>
+  <sky-docs-demo-page-summary>
+    <!-- Page summary goes here -->
```

Comment:
Need a page summary. My initial suggestion is `Selection boxes present users with choices or questions that they must answer before they proceed with one-time processes.`

Reply:
Sounds like a good first guess. We can chew on it for a little bit.